lib/services/manager_api.dart

```diff
@@ -5,6 +5,7 @@ import 'package:device_info_plus/device_info_plus.dart';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_i18n/widgets/I18nText.dart';
+import 'package:fluttertoast/fluttertoast.dart';
```

Comment:
You may instead want to import `package:revanced_manager/services/toast.dart` instead, as `showBottom` is not part of `fluttertoast`.